src/twitch.js

```diff
@@ -1,12 +1,13 @@
 const request = require('./bin/request.js');
 const url = require('url');
 const Promise = require('bluebird');
+const qs = require('querystring');
```

Comment:
should we add `query-string` to the `package.json` dependencies group?

I think you meant to use [`query-string`](https://github.com/sindresorhus/query-string#usage) also instead